src/PowerShellEditorServices/Services/TextDocument/ScriptFileMarker.cs

```diff
@@ -110,19 +110,17 @@ public class ScriptFileMarker
         }
         private static string GetIfExistsString(PSObject psobj, string memberName)
         {
-            if (psobj.Members.Match(memberName).Count > 0)
+            if (psobj.Members.Match(memberName).Count == 0)
             {
-                return psobj.Members[memberName].Value != null ? (string)psobj.Members[memberName].Value : "";
-            }
-            else
-            {
-                return "";
+                return string.Empty;
             }
+
+            return psobj.Members[memberName].Value as string ?? string.Empty;
```

Comment:
won't `.Value` throw?

Reply:
That's [the `Value` property on a `PSMemberInfo`](https://docs.microsoft.com/en-us/dotnet/api/system.management.automation.psmemberinfo.value?view=pscore-6.2.0#System_Management_Automation_PSMemberInfo_Value). But it looks like we don't use this method so I've gotten rid of it